project/Build.scala

```diff
@@ -5,9 +5,9 @@ import sbt._
 import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
 
 object Versions {
-  val Scala211 = "2.11.11"
-  val Scala212 = "2.12.10"
-  val Scala213 = "2.13.1"
+  val Scala211 = "2.11.12"
```

Comment:
spire is not published for 2.11. 

https://mvnrepository.com/artifact/org.typelevel/spire

So, i guess it's time to drop 2.11 suppirt.

Reply:
Spire is used only in tests. I tweaked the build to skip tests for Scala 2.11 in case the author wants to maintain Scala 2.11 / Scala Native.